config/config.go

```diff
@@ -397,6 +398,9 @@ func NewConfig() *Config {
 	flag.Var(cfg.ForwardedHeadersList, "forwarded-headers", "comma separated list of headers to add to the incoming request before routing\n"+
 		"X-Forwarded-For sets or appends with comma the remote IP of the request to the X-Forwarded-For header value\n"+
 		"X-Forwarded-Host sets X-Forwarded-Host value to the request host\n"+
+		"X-Forwarded-Method sets X-Forwarded-Method value to the request method\n"+
+		"X-Forwarded-Uri sets X-Forwarded-Uri value to the requestURI\n"+
+		"X-Original-URL sets X-Original-URL to the absolute original request URL, requires X-Forwarded-Proto to be set\n"+
```

Comment:
Should it be `Uri` and `Url` or `URI` and `URL` maybe?

Reply:
We use `http.CanonicalHeaderKey` in `copyHeader()`, so this would be a breaking change to change this behavior if someone implicitly relies on it.

Reply:
Yeah, the actual header names would be canonical
```
X-Forwarded-Uri: /foo
X-Original-Url: https://127.0.0.1:9090/foo
```
so lets make flag names canonical as well fro consistency.

Reply:
X-Original-Url was dropped, so nothing to do